h2o-algos/src/main/java/hex/deeplearning/DeepLearning.java

```diff
@@ -479,6 +479,8 @@ public final DeepLearningModel trainModel(DeepLearningModel model) {
         if (!_parms._quiet_mode) {
           Log.info("==============================================================================================================================================================================");
           if (stop_requested()) {
+            warn("_max_runtime_secs", "Deep Learning model training was interrupted.  " +
```

Comment:
This can also happen when the user cancels the job explicitly. We should probably only show this warning if there was a timeout (timeout() function returns true).